KeelhaulDemo/ViewController.swift

```diff
@@ -7,10 +7,11 @@ class ViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    let token = "<#API Key#>"
+    let token = "AlV4w1FqkAORqPDaS5OUbg"
```

Comment:
I realized that it's better to have a wrong key and be able to see the demo in action that having to bother with filling this up every time you do git operations.